examples/http/kubernetes_http.js

```diff
@@ -0,0 +1,390 @@
+                label: "Authentication attempts: " + d.desc.result,
+                execute: function () { return parseFloat(d.value); },
+                type: D.valueType.RATE
+            });
+        });
+}
+
+/**
+ * fill the config variable with apiserver_current_inflight_requests
+ * @param {[any]} data all the data parsed from the http response body
+ */
+function fillConfigCurrInflightReq(data) {
+    data
+        .filter(function (d) { return d.key == "apiserver_current_inflight_requests"; })
+        .forEach(function (d) {
+            config.push({
+                uid: ("current_inflight_requests_" + d.desc.request_kind).substring(0, 50),
+                label: "Requests current: " + d.desc.request_kind,
+                execute: function () { return parseFloat(d.value); },
+            });
+        });
+}
+
+/**
+ * fill the config variable with workqueue_adds_total and workqueue_depth
+ * @param {[any]} data all the data parsed from the http response body
+ */
+function fillConfigWorkqueue(data){
+    var data1 = data.filter(function (d) { return d.key == "workqueue_adds_total"; });
+    var data2 = data.filter(function (d) { return d.key == "workqueue_depth"; });
+    data1.forEach(function(d1){
+        var name = d1.desc.name;
+        var d2 = data2.filter(function(d){ return d.desc.name == name;});
+        config.push({
+            uid: ("workqueue_adds_total_" + name).substring(0, 50),
+            label: name + " Workqueue adds total",
+            execute: function () { return parseFloat(d1.value); },
+            type: D.valueType.RATE,
+        });
+        if(d2.length){
+            d2 = d2[0];
+            config.push({
+                uid: ("workqueue_depth_" + name).substring(0, 50),
+                label: name + " Workqueue depth",
+                execute: function () { return parseFloat(d2.value); }
+            });
+        }
+    });
+
+}
+
+
+/**
+ * 
+ * @param {string} number to convert to float
+ * @returns float number
+ */
+function toFloat(number){
+    if(number == "+Inf"){
+        return Infinity;
+    }else{
+        return parseFloat(number);
+    }
+}
+
+/**
+ * 
+ * @param {number} number multiplier
+ * @returns the value muliplied by the number
+ */
+function multiplier(number){
+    return function(value){
+        return value*number;
+    };
+}
+
+var fillConfigFns = [
+    fillConfigAuthUser, 
+    fillConfigAuthAttempt,
+    fillConfigCurrInflightReq,
+    fillConfigWorkqueue
+];
+
+/**
+ * 
+ * @param {[object]} data array of objects 
+ * @returns list of domotz variables
+ */
+function extract(data) {
+    return config.map(function (c) {
+        var result;
+        if (Array.isArray(c.execute)) {
+            result = c.execute.reduce(function (a, b) { return b(a); }, data);
+        } else if (typeof (c.execute) == "function") {
+            result = c.execute(data);
+        }
+        return _var(c.uid, c.label, result, c.unit, c.type);
```

Comment:
since the types are number/rate make sure the result is always a number
either change to 
result || 0

or skip the variable creation in the first place (+ logging a failure)